src/app/utils.spec.tsx

```diff
@@ -196,7 +196,7 @@ describe('mergeRefs', () => {
 describe('remsToPx', () => {
 
   it('converts based on body font size', () => {
-    assertDocument().body.style.fontSize = '14px';
+    assertDocument().documentElement!.style.fontSize = '14px';
```

Comment:
can this 14px be a variable?

Reply:
i try not to use variables in tests unless it really saves a lot of space